### ISSUE TITLE
v1.8.1 disable filters problem

### ISSUE THREAD

**Issue by solariz** [butter-cookie-kitkat/ButterQuestTracker#40]
in 1.8.1 I clicked the disable filters button. But I can't turn it off anymore. When I want to turn it of this error occurs:

```
ClassicCodex-1.2.2\quest.lua:429: Usage: GetQuestLogTitle(index)
(tail call): ?:
<in C code>: in function `GetQuestLogTitle'
ClassicCodex-1.2.2\quest.lua:429: in function `RemoveQuestWatch'
ButterQuestTracker-1.8.1\ButterQuestTracker.lua:643: in function `ResetOverrides'
ButterQuestTracker-1.8.1\Options.lua:161: in function <Interface\AddOns\ButterQuestTracker\Options.lua:159>
<in C code>: ?
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:51: in function <...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:49>:
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:843: in function <...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:664>:
<in C code>: ?
Masque-8.2.7-Classic\Libs\AceGUI-3.0\AceGUI-3.0.lua:72: in function <...terface\AddOns\Masque\Libs\AceGUI-3.0\AceGUI-3.0.lua:70>
Masque-8.2.7-Classic\Libs\AceGUI-3.0\AceGUI-3.0.lua:290: in function `Fire'
...ic\Libs\AceGUI-3.0\widgets\AceGUIWidget-CheckBox.lua:68: in function <...ic\Libs\AceGUI-3.0\widgets\AceGUIWidget-CheckBox.lua:57>:
```

german client

**Comment by cecilia-sanare** [comment]
Figured out what the issue is, pushing out 1.8.3 now with a fix.

**Comment by cecilia-sanare** [comment]
This issue should now be resolved in [1.8.3](https://www.curseforge.com/wow/addons/butter-quest-tracker/files/2799201), let me know if you run into any issues!

**Comment by solariz** [comment]
works, thanks
